stdlib/public/core/StringUTF16View.swift

```diff
@@ -572,7 +572,6 @@ extension String.UTF16View {
       _internalInvariant(readIdx < readEnd)
 
       var utf16I = 0
-      let utf16End: Int = remaining
```

Comment:
Same comment here: `utf16End` illustrates the relationship of this value to `utf16Len` and `utf16I` that cannot be easily deduced at the point of use when using the name `remaining`.